pkg/antctl/raw/set/flow-aggregator/command.go

```diff
@@ -0,0 +1,167 @@
+		Use:     "flow-aggregator",
+		Short:   "Update ClickHouse/IPFIX Flow Collector parameters used in Flow Aggregator",
+		Example: example,
+		RunE:    updateRunE,
+	}
+	mutators = map[string]FlowAggregatorConfigMutator{
+		"clickHouse.enable": func(c *flowaggregatorconfig.FlowAggregatorConfig, value string) error {
+			return setBoolOrFail(&c.ClickHouse.Enable, value)
+		},
+		"clickHouse.database": func(c *flowaggregatorconfig.FlowAggregatorConfig, value string) error {
+			return setStringOrFail(&c.ClickHouse.Database, value)
+		},
+		"clickHouse.databaseURL": func(c *flowaggregatorconfig.FlowAggregatorConfig, value string) error {
+			return setStringOrFail(&c.ClickHouse.DatabaseURL, value)
+		},
+		"clickHouse.debug": func(c *flowaggregatorconfig.FlowAggregatorConfig, value string) error {
+			return setBoolOrFail(&c.ClickHouse.Debug, value)
+		},
+		"clickHouse.compress": func(c *flowaggregatorconfig.FlowAggregatorConfig, value string) error {
+			if c.ClickHouse.Compress == nil {
+				c.ClickHouse.Compress = new(bool)
+			}
+			return setBoolOrFail(c.ClickHouse.Compress, value)
+		},
+		"clickHouse.commitInterval": func(c *flowaggregatorconfig.FlowAggregatorConfig, value string) error {
+			return setCommitIntervalOrFail(&c.ClickHouse.CommitInterval, value)
+		},
+		"flowCollector.enable": func(c *flowaggregatorconfig.FlowAggregatorConfig, value string) error {
+			return setBoolOrFail(&c.FlowCollector.Enable, value)
+		},
+		"flowCollector.address": func(c *flowaggregatorconfig.FlowAggregatorConfig, value string) error {
+			return setStringOrFail(&c.FlowCollector.Address, value)
+		},
+	}
+	return Command
+}
+
+func updateRunE(cmd *cobra.Command, args []string) error {
+	kubeconfig, err := raw.ResolveKubeconfig(cmd)
+	if err != nil {
+		return err
+	}
+	k8sClient, _, err := raw.SetupClients(kubeconfig)
+	if err != nil {
+		return fmt.Errorf("failed to create clientset: %w", err)
+	}
+	configMapName := os.Getenv("FA_CONFIG_MAP_NAME")
+	configMap, err := util.GetFAConfigMap(k8sClient, configMapName)
+	if err != nil {
+		return err
+	}
+	var flowAggregatorConf flowaggregatorconfig.FlowAggregatorConfig
+	if err := yaml.Unmarshal([]byte(configMap.Data["flow-aggregator.conf"]), &flowAggregatorConf); err != nil {
+		return err
+	}
+	for _, query := range args {
+		pair := strings.Split(query, "=")
+		if len(pair) != 2 {
+			return fmt.Errorf("query should contains exactly one '='")
+		}
+		mutatorFn, ok := mutators[pair[0]]
+		if !ok {
+			return fmt.Errorf("unknown configuration parameter, please check antctl set -h")
+		}
+		if err := mutatorFn(&flowAggregatorConf, pair[1]); err != nil {
+			return err
+		}
+	}
+	// marshal back the changed parameters to configmap
+	b, err := yaml.Marshal(&flowAggregatorConf)
+	if err != nil {
+		return err
+	}
+	configMap.Data["flow-aggregator.conf"] = string(b)
+	if _, err := k8sClient.CoreV1().ConfigMaps("flow-aggregator").Update(context.TODO(), configMap, metav1.UpdateOptions{}); err != nil {
```

Comment:
rather than hardcoding the Namespace name (`flow-aggregator`), let's also retrieve it through the environment / Pod downward API (like we do for the ConfigMap name).

Reply:
Added